A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/EICAS_Common.js

```diff
@@ -77,23 +77,23 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
         }
     }
     refreshGrossWeight(_force = false) {
-        let gw = 0;
         let isInMetric = BaseAirliners.unitIsMetric(Aircraft.A320_NEO);
-        if (isInMetric) {
-            gw = Math.round(SimVar.GetSimVarValue("TOTAL WEIGHT", "kg"));
-            if (this.gwUnit)
-                this.gwUnit.textContent = "KG";
-        }
-        else {
-            gw = Math.round(SimVar.GetSimVarValue("TOTAL WEIGHT", "lbs"));
-            if (this.gwUnit)
-                this.gwUnit.textContent = "LBS";
+        let unit = isInMetric ? "kg" : "lbs";
```

Comment:
Any variable that isn't changed after it's declaration should be a `const` instead of a `let` so we know it isn't changed somewhere down the line.

Reply:
ok, I'm working on it and push new commit soon.